tests/Tokenizer/Analyzer/GotoLabelAnalyzerTest.php

```diff
@@ -83,6 +83,17 @@ function A(){}
                 ',
                 [10],
             ],
+            'handle goto labels 5' => [
```

Comment:
Can we drop the indices? The don't add any information.

Reply:
Done.